Cognite.Simulator.Utils/ModelParsingInfo.cs

```diff
@@ -23,6 +25,16 @@ public class ModelParsingInfo
         /// </summary>
         public string StatusMessage { get; set; }
```

Comment:
```suggestion
        public string? StatusMessage { get; set; }
```
need to make it nullable, otherwise there are warnings below